### ISSUE TITLE
JAWS does not scroll correctly in scrollable elements when using virtual focus

### ISSUE THREAD

**Issue by jdrush89** [FreedomScientific/VFO-standards-support#607]
### Summary

> Brief description of the issue. Please list any specific steps.
> 
> Example:
> 1. Start JAWS
> 1. Go to https://codesandbox.io/s/intelligent-resonance-ujiymw
> 2. tab to the table in the right pane, or use 't' to focus the table
> 3. use ctrl + alt + down arrow to navigate JAWS's virtual focus through the rows

### Expected result

> The tablescrolls as the virtual focus moves out of the current viewport

### Actual result

> The list does not scroll unless the virtual focus hits bounds of the browser window. This means JAWS users cannot reach elements
in virtualized lists/tables which rely on the scroll position in the list to load more items.  NVDA and Narrator do not have this problem, they scroll the table when their virtual focus reaches the edge of the table's visible bounds. 

### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...
https://codesandbox.io/s/intelligent-resonance-ujiymw

#### JAWS version and build number
Professional Edition Version 2022.2110.36 ILM

 
#### Operating System and version
Windows 10 Enterprise Build 19041.vb_release.191206-1406

#### Browser and version:
Chrome Version 99.0.4844.51 (Official Build) (32-bit)

**Comment by MadanKumar7** [comment]
Is this issue still open. I am also facing the same issue.

**Comment by patrick-va** [comment]
Hello,
Just like @amycole501 mentioned above, we're using UWWDS scrollable tables and are experiencing the same issue with JAWS 2025, too. 
Please provide an update at your earliest convenience, thanks!
https://github.com/uswds/uswds/issues/5828
